CHANGELOG.md

```diff
@@ -7,20 +7,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [3.0.0-beta.0] - 2022-03-16
+## [3.0.0-beta.0] - 2022-03-15
+
+### Fixed
```

Comment:
nit: Why is Fixed up here? :D

Reply:
Component library puts Fixed at the top

Reply:
Seem like a bit random and is not the same as JS SDK but oh well.

Reply:
I don't disagree, but we'll need to write a script to update the changelog and modify the release script which should be in a different PR I feel.